e2e-tests/playwright/e2e/github-happy-path.spec.ts

```diff
@@ -114,7 +114,7 @@ test.describe.serial('GitHub Happy path', () => {
     await backstageShowcase.verifyPRRows(closedPRs, 0, 5);
   });
 
-  test('Click on the arrows to verify that the next/previous/first/last pages of PRs are loaded', async () => {
+  test.skip('Click on the arrows to verify that the next/previous/first/last pages of PRs are loaded', async () => {
```

Comment:
Please comment the JIRA bug for reference here.

Reply:
Sure, makes sense. I added it. Thank you